domain.yml

```diff
@@ -35,15 +44,26 @@ slots:
     type: unfeaturized
   requested_slot:
     type: unfeaturized
+
 responses:
-  utter_out_of_scope:
-  - text: Sorry, I'm not sure how to respond to that. Type "help" for assistance.
```

Comment:
I think this was removed by mistake - there is still an intent and stories for it?

Reply:
yes, added back.